Modules/Core/TestKernel/include/itkTestingMacros.h

```diff
@@ -38,7 +38,7 @@ namespace itk
 #define EXERCISE_BASIC_OBJECT_METHODS \
   static_assert(false, "Replace EXERCISE_BASIC_OBJECT_METHODS with ITK_EXERCISE_BASIC_OBJECT_METHODS")
 #define TRY_EXPECT_EXCEPTION static_assert(false, "Replace TRY_EXPECT_EXCEPTION with ITK_TRY_EXPECT_EXCEPTION")
-#define TRY_EXPECT_NO_EXCEPTION static_assert(fasle, "Replace TRY_EXPECT_NO_EXCEPTION with ITK_TRY_EXPECT_NO_EXCEPTION")
+#define TRY_EXPECT_NO_EXCEPTION static_assert(false, "Replace TRY_EXPECT_NO_EXCEPTION with ITK_TRY_EXPECT_NO_EXCEPTION")
```

Comment:
I found this old bug, it's in a deprecated section. Should i revert this?

Reply:
It will not compile either way, which is the intent. Let's keep the correct spelling.